rust/syntect-plugin/Makefile

```diff
@@ -6,22 +6,27 @@ XI_PLUGIN_DIR ?= $(XI_CONFIG_DIR)/plugins
 
 lang_configs = assets/*.toml
 
-out/syntect: $(lang_configs) xi-syntect-plugin
+out/syntect: assets $(lang_configs) xi-syntect-plugin
```

Comment:
I think we don't actually want this here; we don't regenerate the assets each time somebody installs the plugin, but only during development.